### DISCUSSION TITLE
renderAsync or render?

### DISCUSSION THREAD

**Discussion by Mister-Hope** [mermaid-js/mermaid#4091] [Q&A] [answered]
From api jsdocs, `renderAsync` is marked as deprecated, and a msg telling that it's going to removed in v10, while `render` is not.

However, recent diagrams like flowchart v2, mindmap and timeline are all seems to be async, and doesnot work in latest release 9.4.0.

This is confusing me as maintainers are creating async diagrams while marking asyncRender deprecated. Can any team member explain this?

**Comment by sidharthv96** [comment, marked as answer]
Hmm, I agree it's a bit confusing. 
You can use `renderAsync` for all diagrams now.
The change coming in v10 is that the current `render` will be replaced by `renderAsync`, so `renderAsync` function will technically be removed in v10.
